### PR TITLE
Create Content for Meta Transactions

### PR DESCRIPTION
**Pull request by DannyS03**
- Guide that explains meta transactions
- Removed pages on the specific solutions as the gsn guide is a copy of the documentation available on the openzeppelin docs and the biconomy guide isn't necessary in its current form

### CODE REVIEW

File: docs/develop/meta-transactions/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+The signed transaction parameters are passed onto a secondary network, which acts as a relayer. 
+While there are different schemes for this, relayers would generally choose which transactions are worth 
+submitting by validating the transaction (e.g., being relevant to the dApp). Upon validation, the relayer 
+will wrap the request (the signed message) into an actual transaction (which means paying the gas fee) and 
+broadcast it to the network, where the contract unwraps the transaction by validating the original signature 
+and executes it on behalf of the user.
+
+Note: The words meta and batch may be analogous to some. To clarify: a meta transaction is different from 
+a batch transaction, where a batch transaction is a transaction that can send multiple transactions at once 
+and are then executed from a single sender (single nonce specified) in sequence.
```
Comment (cerberushades) [inline docs/develop/meta-transactions/getting-started.md:66]: that could be an admonition box

File: docs/develop/meta-transactions/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+In summary, meta transactions are a design pattern where:
+
+A user (sender) signs a request with their private key and sends it to a relayer
+The relayer wraps the request into a tx and sends it to a contract 
+The contract unwraps the tx and executes it
```
Comment (cerberushades) [inline docs/develop/meta-transactions/getting-started.md:72]:
```suggestion
- A user (sender) signs a request with their private key and sends it to a relayer
- The relayer wraps the request into a tx and sends it to a contract 
- The contract unwraps the tx and executes it
```

File: docs/develop/meta-transactions/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+## Try Them Out
+
+Assuming your familiarity with the different approaches you can take to integrate meta-transactions in your 
+dApp, and depending on whether you're migrating to meta-transactions or building fresh dApp on using it.
+
+To integrate your dApp with Meta Transactions on Polygon, you can choose to go with either the following two 
+relayers or spin up a custom solution:
+
+1. [Biconomy](meta-transactions-biconomy.md)
+2. [Gas Station Network (GSN)](meta-transactions-gsn.md)
```
Comment (cerberushades) [inline docs/develop/meta-transactions/getting-started.md:109]: both links appear broken to me

**Review by cerberushades — CHANGES_REQUESTED** [review]

File: docs/develop/meta-transactions/getting-started.md (hunk 3 above)
Comment (cerberushades) [reply docs/develop/meta-transactions/getting-started.md:109]: I think removing ".md" might solve it

File: docs/develop/meta-transactions/getting-started.md (hunk 2 above)
Comment (DannyS03) [reply docs/develop/meta-transactions/getting-started.md]: Updated

File: docs/develop/meta-transactions/getting-started.md (hunk 3 above)
Comment (DannyS03) [reply docs/develop/meta-transactions/getting-started.md]: Addressed

File: docs/develop/meta-transactions/getting-started.md (hunk 1 above)
Comment (DannyS03) [reply docs/develop/meta-transactions/getting-started.md]: Addressed

**Comment by DannyS03** [comment]
Removed pages on the specific solutions as the gsn guide is a copy of the documentation available on the openzeppelin docs and the biconomy guide isn't necessary in its current form

**Review by cerberushades — APPROVED** [review]

**Comment by cerberushades** [comment]
@DannyS03 I think you need to delete the following lines : - develop/meta-transactions/meta-transactions-biconomy
- develop/meta-transactions/meta-transactions-gsn

**Comment by DannyS03** [comment]
@cerberushades I removed the most recent commit as it conflicts with the latest changes and build fails. Building is successful locally and passing CI with the changes in `dd5989c`

**Pull request merged**